tests/phpunit/admin/class-wc-stripe-settings-controller-test.php

```
@@ -110,4 +110,98 @@ public function test_add_buttons_action_is_called_on_order_admin_page() {
+			unset( $current_tab, $current_section );
+		}
+	}
+
+	public function provide_test_admin_scripts_checkout_sessions_country_restrictions(): array {
+		return [
+			'US account + feature available'              => [ 'US', true, true ],
+			'DE account (EEA) + feature available'        => [ 'DE', true, false ],
+			'US account + feature unavailable'            => [ 'US', false, false ],
+			'CA account + feature available (not in EEA)' => [ 'CA', true, true ],
+		];
+	}
```

PHPMD (2.15.0)

[error] The method provide_test_admin_scripts_checkout_sessions_country_restrictions is not named in camelCase. (undefined)

(CamelCaseMethodName)